docs/developers/release-checklist.md

```diff
@@ -4,19 +4,14 @@
-7. Test `brew upgrade ddev` both on macOS and Linux and make sure ddev is the right version and behaves well
-8. Test the Windows installer and confirm it's signed correctly
-9. Update the release page with specifics about the current release
-10. Publish the release (unmark it as "prerelease")
-11. Download the ddev_chocolatey tarball and extract it. cd into the extraction directory and push it to chocolatey with `docker run --rm -v $PWD:/tmp/chocolatey -w /tmp/chocolatey linuturk/mono-choco push -s https://push.chocolatey.org/ --api-key=choco-apikey-a720-7890909913f7`  (Although this ought to be done by the release build process on CircleCI it's not successful as of v1.7.1.)
-12. On [ReadTheDocs](https://readthedocs.org/projects/ddev/builds) click the button to "build version" "latest".  Then on [versions](https://readthedocs.org/projects/ddev/versions/) page make sure that "stable" reflects the hash of the new version.
-
-## Creating a Release
-
-Just use the github UI to create a release on the releases page. Add notes as needed, then the CircleCI build process will overwrite the release info on the build.
+6. Update the `ddev` homebrew formulas (ddev-edge and ddev) as necessary, <https://github.com/drud/homebrew-ddev> and <https://github.com/drud/homebrew-ddev-edge,> with the source .tar.gz and SHA checksum of the tarball and the bottle builds and tarballs. The bottles and checksums for macOS (sierra) and x86_64_linux are built and pushed to the release page automatically by the CircleCI release build process. Test `brew upgrade ddev` both on macOS and Linux and make sure ddev is the right version and behaves well.
+7. Download the ddev_chocolatey tarball and extract it. It should be available on a Windows machine or VM. (It can be network-mounted into a Windows VM). cd into the extraction directory and push it to chocolatey with `choco push -s https://push.chocolatey.org/ --api-key=choco-apikey-a720-asome-api-key` Before 2020-02-03 this coudl be done from Linux or macOS with mono-choco, but doesn't work any more due to [mono-choco#20](https://github.com/Linuturk/mono-choco/issues/20) ). After the push responses have come back, install with `choco install -y --pre ddev --version <version>` and verify correct behavior.
+8. Publish the release to the ddev [AUR repository](https://aur.archlinux.org/packages/ddev-bin/). The README.md in the AUR git repo (`ssh://aur@aur.archlinux.org/ddev-bin.git` or `https://aur.archlinux.org/ddev-bin.git`) has instructions on how to update, including how to do it with a Docker container, so it doesn't have to be done on an ArchLinux or Manjaro VM)
```

Comment:
https://aur.archlinux.org/ddev-bin.git won't be useful to anyone. IMO, just include the ssh URL here.

Also, unbalenced parenthesis at the end of the line.

Reply:
Leaving the https URL in there because it's a way to get to the instructions if you don't have ssh set up. Fixed the paren.